spec/v3/services/job/find_spec.rb

```diff
@@ -23,7 +23,8 @@
         "read"            => true,
         "cancel"          => false,
         "restart"         => false,
-        "debug"           => false },
+        "debug"           => false,
+        "delete_log"      => false },
```

Comment:
This is always a pain. Maybe we can talk after feature freeze about how to avoid this every time.